test/emqtt_quic_SUITE.erl

```diff
@@ -99,6 +99,10 @@ groups() ->
     ].
 
 init_per_suite(Config) ->
+    DataDir = filename:join([test_dir(Config), "certs"]),
+    emqtt_test_lib:gen_ca(DataDir, "quic_ca"),
+    emqtt_test_lib:gen_host_cert("quic", "quic_ca", DataDir),
+    emqtt_test_lib:gen_host_cert("quic", "quic_ca", DataDir),
```

Comment:
Is this repetition intentional?  It seems like this will just overwrite the files generated in the previous run.